### PR TITLE
Update functionMap.php to fix return types and parameters

### PR DESCRIPTION
**Pull request by vindic**
Fix for https://github.com/phpstan/phpstan/issues/12148#issue-2704647597 

Fixing the signatures for the following functions :

apcu_cache_info()
apcu_sma_info()

And another one

strval()

setting the parameter value to mixed as defined https://www.php.net/manual/en/function.strval.php 

Code snippet that reproduces the problem :
https://phpstan.org/r/f3e4b8de-f919-44d2-8b08-b979c5240693

### CODE REVIEW

**Comment by ondrejmirtes** [comment]
Don't change the strval signature, it's actually correct. To find out why, search the issue tracker.

File: resources/functionMap.php [hunk 1]
```diff
@@ -12088,7 +12088,7 @@
 'strtoupper' => ['uppercase-string', 'str'=>'string'],
 'strtr' => ['string', 'str'=>'string', 'from'=>'string', 'to'=>'string'],
 'strtr\'1' => ['string', 'str'=>'string', 'replace_pairs'=>'array'],
-'strval' => ['string', 'var'=>'__stringAndStringable|int|float|bool|resource|null'],
+'strval' => ['string', 'var'=>'mixed'],
```
Comment (staabm) [inline resources/functionMap.php]: I think it was correct before, see e.g. https://github.com/phpstan/phpstan/issues/9295#issuecomment-1542186125

Comment (vindic) [reply resources/functionMap.php]: Ok, so the PHP documentation is incorrect, next time I will check the issue tracker first.

I have reverted and commit it again.

**Comment by ondrejmirtes** [comment]
Thank you.

**Pull request merged**